src/data/set/finite.lean

```diff
@@ -226,6 +226,12 @@ lemma infinite.exists_subset_card_eq {s : set α} (hs : infinite s) (n : ℕ) :
   ∃ t : finset α, ↑t ⊆ s ∧ t.card = n :=
 ⟨((finset.range n).map (hs.nat_embedding _)).map (embedding.subtype _), by simp⟩
 
+lemma infinite.nonempty {s : set α} (h : s.infinite) : s.nonempty :=
+begin
+  haveI := set.infinite_coe_iff.2 h,
+  exact set.nonempty_of_nonempty_subtype
+end
```

Comment:
```
let a := infinite.nat_embedding s h 37 in ⟨a.1, a.2⟩
```
avoids tactic mode (it compiles in the time it takes to open up tactic mode :-) )